index.html

```diff
@@ -4596,78 +4596,80 @@ <h2>Key and Signature Expiration</h2>
-        some time.
+The following considerations might be of use when contemplating the use of
+<a>verification method</a> rotation:
       </p>
 
-      <p>Not all DID Methods support verification method rotation.</p>
-
-      <p>
-        Rotation is a key management process that enables the private cryptographic
-        material associated with an existing verification method to be deactivated
-        or destroyed once a new verification method has been added to the DID
-        Document. Going forward, any new proofs that a controller would have generated
-        using the old cryptographic material can now instead be generated using the new
-        material and can be verified using the new verification method.
-      </p>
-
-      <p>
-        Rotation is a useful mechanism for protecting against verification
-        method compromise, since frequent rotation of a verification method by
-        the controller reduces the value of a single compromised verification method
-        to an attacker. Performing revocation immediately after rotation is
-        useful for verification methods that a controller designates for
-        short-lived verifications, such as those involved in encrypting messages
-        and authentication.
-      </p>
-
-      <p class="note">
-        Higher security environments tend to employ more frequent verification
-        method rotation.
-      </p>
-
-      <p class="note">
-        Frequent rotation of a verification method might be frustrating for
-        parties that are forced to continuously renew or refresh associated
-        credentials.
-      </p>
+      <ul>
+        <li>
+<a>Verification method</a> rotation is a proactive security measure.
+        </li>
+        <li>
+It is considered a best practice to perform <a>verification method</a> rotation
+on a regular basis.
+        </li>
+        <li>
+Higher security environments tend to employ more frequent verification method
+rotation.
+        </li>
+        <li>
+<a>Verification method</a> rotation applies only to the current or latest
+version of a <a>DID document</a>.
+        </li>
+        <li>
+When a <a>verification method</a> has been active for a long time, or used for
+many operations, a controller might wish to perform a rotation.
+        </li>
+        <li>
+Frequent rotation of a <a>verification method</a> might be frustrating for
+parties that are forced to continuously renew or refresh associated credentials.
+        </li>
+        <li>
+Proofs or signatures that rely on <a>verification methods</a> that are not
+present in the latest version of a <a>DID document</a> are not impacted by
+rotation. In these cases, verification software might require additional
+information, such as when a particular <a>verification method</a> was
+expected to be valid as well as access to a <a>verifiable data registry</a>
+containing a historical record, to determine the validity of the proof or
+signature.
```

Comment:
```suggestion
signature. In some <a>DID methods</a> this option may not be available.
```

Reply:
Specifically trying to call out not all methods will support versioning/historical records even if update/rotation is supported.

Reply:
Applied @TallTed 's suggestion to @kdenhartog 's comment.